resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -661,6 +665,33 @@ private[spark] class Client(
       }).toMap
     }).getOrElse(Map.empty[String, String])
   }
+
+  private def checkForUserJarsFilesExistence: Unit = {
+    val conf = new Configuration
+
+    def existsOrAbort(uri: URI): Unit = {
```

Comment:
I'd rather avoid nesting methods here. We can just inline this logic in the case statement of `checkExistence`.